Source/WebCore/rendering/svg/legacy/LegacyRenderSVGContainer.cpp

```diff
@@ -253,7 +253,14 @@ bool LegacyRenderSVGContainer::nodeAtFloatPoint(const HitTestRequest& request, H
             return true;
     }
     
-    // Spec: Only graphical elements can be targeted by the mouse, period.
+    // pointer-events=bounding-box makes it possible for containers to be direct targets.
+    if (style().pointerEvents() == PointerEvents::BoundingBox) {
+        ASSERT(isObjectBoundingBoxValid());
```

Comment:
Why do we `ASSERT(isObjectBoundingBoxValid())`? Is it guaranteed that this function will never be called for `style().pointerEvents() == PointerEvents::BoundingBox` unless `isObjectBoundingBoxValid()` is true?

Reply:
@shallawa - something similar is done in LBSE as well - https://searchfox.org/wubkat/rev/6bcd0944c131d3f3088997a26c691bc8c7e84fc2/Source/WebCore/rendering/svg/SVGBoundingBoxComputation.cpp#150

In short, we need to ensure that bounding-box is not empty plus it also don't contribute back to parent container. Latter, was fixed recently in LegacySVG (https://commits.webkit.org/292411@main).

Reply:
@shallawa - any thoughts?

Reply:
@shallawa - Sorry - I might have misunderstood your question. I have pushed update to return false now in [5bf083c](https://github.com/WebKit/WebKit/commit/5bf083c40b431cf348dd7699ec1c0f24e16126fb).